package com.example.Exceptions;

public class Exceptions {

	public static void main(String[] args) {
		int a = 0;
		int b = 1;
		try{
			int c = b/a;
			System.out.println(c);
		}
		catch(Exception e){
		System.out.println("After Exception1");
		System.out.println("After Exception2");
		}
	}

}
